internal/engine/buildcontrol/build_control_test.go

```diff
@@ -29,6 +30,37 @@ func TestNextTargetToBuildDoesntReturnCurrentlyBuildingTarget(t *testing.T) {
 	f.assertNoTargetNextToBuild()
 }
 
+func TestPodsDisableLocalScheduling(t *testing.T) {
+	f := newTestFixture(t)
+	defer f.TearDown()
+
+	pod1 := f.upsertK8sManifest("pod1")
+	pod2 := f.upsertK8sManifest("pod2")
+	f.upsertLocalManifest("local1")
+
+	f.assertNextTargetToBuild("local1")
+
+	pod1.State.CurrentBuild = model.BuildRecord{StartTime: time.Now()}
+	f.assertNextTargetToBuild("pod2")
+
+	pod2.State.CurrentBuild = model.BuildRecord{StartTime: time.Now()}
+	f.assertNoTargetNextToBuild()
+}
+
+func TestLocalDisablesPodScheduling(t *testing.T) {
```

Comment:
same comment as above (but even more so, because "pod scheduling" is its own thing 😅 )

Reply:
done